code/modules/antagonists/cult/cult_items.dm

```diff
@@ -503,26 +503,40 @@
 	if(SSshuttle.emergency.mode == SHUTTLE_CALL)
 		var/cursetime = 1800
 		var/timer = SSshuttle.emergency.timeLeft(1) + cursetime
+		var/security_num = seclevel2num(get_security_level())
+		var/set_coefficient = 1
+		switch(security_num)
+			if(SEC_LEVEL_GREEN)
+				set_coefficient = 2
+			if(SEC_LEVEL_BLUE)
+				set_coefficient = 1
+			else
+				set_coefficient = 0.5
+		var/surplus = timer - (SSshuttle.emergencyCallTime * set_coefficient)
 		SSshuttle.emergency.setTimer(timer)
+		if(surplus > 0)
+			SSshuttle.block_recall(surplus)
```

Comment:
I feel weird about this recall blocking. Can't you just instantly recall the shuttle, and then block anyone trying to call it for the same amount of time?

Reply:
That feels even weirder and would be a flat out buff since it would require ANOTHER shuttle call and waiting around for some invisible timer to pass. This method gives a clear rule: You can't recall while the timer is above its "start point", so there's no reason to even mess with the comms console if someone curses the shuttle.